README.md

```diff
@@ -18,26 +18,22 @@ like JsonView or XmlView.
 ## Requirements
 
 * CakePHP 3.5.5 or greater
-* PHP 5.4.16 or greater
+* PHP 5.6 or greater
 * Patience
 
 ## Installation
 
 _[Using [Composer](http://getcomposer.org/)]_
 
 ```
-composer require friendsofcake/cakephp-csvview:~3.0
+composer require friendsofcake/cakephp-csvview
 ```
 
 ### Enable plugin
 
-Load the plugin in your app's `config/bootstrap.php` file:
+Load the plugin by running command
 
-	Plugin::load('CsvView');
-
-In CakePHP version 3.5 or newer you should use:
-
-    Plugin::load('CsvView', ['routes' => true]);
+    ./bin/cake plugin load CsvView
```

Comment:
I would just use bin/cake
As for Windows it would be .\ anyway etc, so better to keep it simple.